Makefile

```
@@ -0,0 +1,10 @@
+.PHONY: ifs docker clean
+
+ifs: *.go
+	go build -o docker ./...
+
+docker:
```

Would be good for this to depend on the `ifs` target to make sure that it's actually built

```suggestion
docker: ifs
```